src/input.css

```diff
@@ -26,114 +26,487 @@
   }
 
   .primary-btn--full-width {
-    @apply font-roboto font-extrabold text-white bg-primary hover: bg-light-violet hover:text-primary transition-all duration-500 ease-in-out cursor-pointer w-full h-[56px] flex justify-center items-center rounded-xsm text-paragraph;
+    @apply font-roboto font-extrabold text-white bg-primary hover:bg-light-violet hover:text-primary transition-all duration-500 ease-in-out cursor-pointer w-full h-14 flex justify-center items-center rounded-xsm text-paragraph;
   }
 
   .primary-btn--fill-white {
-    @apply font-roboto font-extrabold text-primary bg-white hover: bg-primary hover:text-white transition-all duration-500 ease-in-out cursor-pointer w-[144px] h-[56px] flex justify-center items-center rounded-xsm text-paragraph;
+    @apply font-roboto font-extrabold text-primary bg-white hover:bg-primary hover:text-white transition-all duration-500 ease-in-out cursor-pointer w-36 h-14 flex justify-center items-center rounded-xsm text-paragraph;
   }
 
   .primary-btn--outline {
     @apply primary-btn--fill-white border-2 border-light-violet;
   }
 
+  .secondary-btn{
+    @apply bg-slate-100 rounded-lg w-36 text-black h-14 border-2;
+  }
+
   .gradient-paragraph {
     @apply text-gradient;
   }
 
-  .paragraph-large {
-    @apply text-[18px];
+  .btn {
+    @apply max-w-5xl h-[42rem] mx-auto mt-10 bg-neutral-50;
+  }
+
+  body {
+    @apply relative w-full h-full font-roboto;
   }
 
-  .btn {
-    @apply w-[1024px] h-[666.57px] mx-auto mt-10 bg-neutral-50;
+  .nav {
+    @apply flex flex-row justify-between items-center px-20 mt-3;
+  }
+
+  .nav__logo {
+    @apply font-bold text-base;
+  }
+
+  .nav__menu {
+    @apply flex flex-row gap-8;
   }
 
   .body-btn {
-    @apply bg-slate-800 rounded-lg w-[144px] text-white h-[56px];
+    @apply bg-slate-800 rounded-lg w-36 text-white h-14;
+  }
+
+  .body-background {
+    @apply absolute w-[33rem] h-[31rem] right-[35rem] top-[20rem] background -z-20;
   }
 
-  .back-secondary {
-    @apply absolute w-[524px] h-[496px] right-[566px] top-[313px] body-background -z-20;
+  .body-background__content {
+    @apply px-20 flex flex-col justify-center items-center mt-20;
   }
 
-  .mail-input {
-    @apply flex justify-between items-center h-[72px] mx-auto bg-white w-[576px] mt-10 rounded-[20px] p-3;
+  .body-background__title {
+    @apply font-roboto font-bold text-6xl leading-tight text-center text-slate-800;
+  }
+
+  .body-background__paragraph {
+    @apply mt-8 font-roboto font-normal text-lg leading-normal text-center;
+  }
+
+  .body-background__mail-input {
+    @apply flex justify-between items-center h-16 mx-auto bg-white w-[36rem] mt-10 rounded-sm p-3;
+  }
+
+
+  .body-background__input-holder {
+    @apply bg-transparent font-roboto;
   }
 
   .body-image {
-    @apply w-[1024px] h-[666.57px] mx-auto mt-10 bg-neutral-50;
+    @apply max-w-5xl h-[42rem] mx-auto mt-10 bg-neutral-50;
+  }
+
+  .body-image__top {
```

Comment:
the name and style content are differents... improved the style or the naming or both :)